presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -253,7 +255,8 @@ it is highly recommended that you set ``hive.s3.use-instance-credentials``
 to ``true`` and use IAM Roles for EC2 to govern access to S3. If this is
 the case, your EC2 instances will need to be assigned an IAM Role which
 grants appropriate access to the data stored in the S3 bucket(s) you wish
-to use.  This is much cleaner than setting AWS access and secret keys in
+to use. Also it's possible to set ``hive.s3.iam-role`` with an IAM Role
+with access to the desired S3 bucket(s).  This is much cleaner than setting AWS access and secret keys in
```

Comment:
What do you mean by `desired S3 buckets`? IIUC when this config property is set that role will be assumed for accessing any S3 bucket.  Maybe we should say sth like:

```
...
allows EC2 to automatically rotate credentials on a regular basis without
any additional work on your part. It's also possible to configure an IAM 
role with ``hive.s3.iam-role`` that will be assumed for accessing any S3 bucket.
```